docs/docs/deploy/google-cloud-run.md

```diff
@@ -61,6 +61,8 @@ Running this command will do three things:
 - Build the Docker image in [Cloud Build](https://cloud.google.com/build) and upload it to [Artifact Registry](https://cloud.google.com/artifact-registry)
 - Deploy the image to the specified Cloud Run service
 
+There is an ongoing [issue](https://github.com/google/gvisor/issues/7331) that is preventing Dart Unix Sockets from working correctly in the gen1 version of the Cloud Run environment. In that case, it is recommended to deploy to gen2 by adding `--execution-environment=gen2`
+
```

Comment:
I've unresolved this since it seems that the last section hasn't been addressed yet 🙌 

> Could we also move this "caution bloc" directly above "Running this command will do three things:"? That way it directly follows the snippet of code where it has to be included 🙌

Reply:
@alestiago should be done 👍 let me know if its ok for you